webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const TerserPlugin = require('terser-webpack-plugin')
+const webpack = require('webpack')
+const dotenv = require('dotenv')
```

Comment:
Hmm chyba nigdzie tego nie wykorzystujesz?

Reply:
`GOOGLE_API_KEY: JSON.stringify(process.env.GOOGLE_API_KEY)` to wymaga dotenva

Reply:
Ale samego importu? Dziwne :/

Reply:
No ale tak jest zawsze nie? process.env jest dotenvowy, tak samo w czesci nodeowej jak masz process.env to dajesz require('dotenv') no nie? :p